spec/plaid_spec.rb

```diff
@@ -49,6 +49,13 @@
         end
       end
     end
+
+    describe '.delete' do
+      it 'responds with a status code of 200' do
```

Comment:
are there any unhappy cases, where the delete would fail? eg token doesn't exist?